src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -33,12 +60,14 @@ public Object getTimestampProperty(String entity, DatastoreService datastore) {
         try {
             pq = datastore.prepare(query);
         } catch(NullPointerException e) {
+            log.severe("Null datastore");
             return null;
```

Comment:
Does this mean that instead of the return statement I should just have the log.severe() part? Won't that make the tests fail though?

Reply:
No, my suggestion would be to not catch the NullPointerException in the first place, and instead just let the program crash. But I think Java makes it difficult to do this, so it might not be worth worrying about for now.